src/language/tekster_nn.ts

```diff
@@ -682,14 +686,22 @@ export default {
   'utdanning.spm.mål': 'What is your goal for taking this education?',
   'utdanning.alert-tittel.mål':
     'You must provide documentation of the education you are taking or are going to take',
-  'utdanning.alert-beskrivelse.mål':
+  'utdanning.alert-beskrivelse.mål.skolepenger':
+    'The documentation must indicate:<ul>' +
+    '<li>the name of the place of study</li>' +
+    '<li>the name of the course or education</li>' +
+    '<li>how much you are going to study</li>' +
+    '<li>the period in which you are going to study</li>' +
+    '</ul><br/><br/>' +
+    'The documentation must clearly show who it concerns. ',
+  'utdanning.alert-beskrivelse.mål.overgangsstønad':
     'The documentation must indicate:<ul>' +
     '<li>the name of the place of study</li>' +
     '<li>the name of the course or education</li>' +
     '<li>how much you are going to study</li>' +
     '<li>the period in which you are going to study</li>' +
     '</ul><br/><br/>' +
-    'The documentation must show clearly to whom it pertains. ' +
+    'The documentation must clearly show who it concerns. ' +
```

Comment:
Dette er kanskje flisespikkeri, men: 
pertains leser jeg som - hvem "dokumentasjonen dokumenterer", mens concerns kanskje er litt videre - kan også bety den det angår, er relevant for, hvem som er leser e.l (men stort spørsmålstegn her - jeg er ikke engelskekspert)